### ISSUE TITLE
conflict with sinon-chai

### ISSUE THREAD

**Issue by Offirmo** [gakimball/chai-fetch-mock#3]
Previous chai plugin sinon-chai https://www.chaijs.com/plugins/sinon-chai/ also defines `spy.should.have.been.called`

When testing my sinon spy, I get the error: `AssertionError: Cannot check if a route has been called without route() in the assertion.`

```
const spy = sinon.spy()

expect(spy).to.not.have.been.called
```

**Comment by gakimball** [comment]
I reworked the plugin so that its own methods only kick in if you call `route()` first; otherwise, another plugin's methods will run instead.

I used the below script to test it. Note that you have to apply chai-fetch-mock _after_ sinon-chai.

Can you install this and let me know if it solves your problem? If so, I can publish a new version of the library.

```
npm install gakimball/chai-fetch-mock#ab941b81b7eb215f15cb630114e74822380884b6 --save-dev
```

```js
const chai = require('chai');
const sinon = require('sinon');
const sinonChai = require('sinon-chai');
const fetchMock = require('fetch-mock');
const chaiFetchMock = require('.');

chai.use(sinonChai);
chai.use(chaiFetchMock);

fetchMock.get('/test', {test: true});

const spy = sinon.spy();

spy();

fetch('/test').then(() => {
  // `called` uses chai-fetch-mock here
  chai.expect(fetchMock).route('/test').called;

  // `called` uses sinon-chai here
  chai.expect(spy).called;
})
```

**Comment by Offirmo** [comment]
Did a quick test, seems to work 👍

**Comment by gakimball** [comment]
Great, thank you! I've published v3.0.0 with the fix.

**Comment by Offirmo** [comment]
Thanks @gakimball , you're amazing!
